### PR TITLE
update tests for new email sending flow

### PR DESCRIPTION
**Pull request by botany-lead**
update tests for new email sending flow

### CODE REVIEW

**Comment by botany-lead** [comment]
what do you think @botany-demo

File: readme.md [hunk 1]
```diff
@@ -37,3 +37,5 @@ add additional user config options
 consistent ordering of user states
 
 cleanup documentation in README file
+
+update tests for new email sending flow
```
Comment (botany-demo) [inline readme.md:41]: this looks good so far

Comment (botany-lead) [reply readme.md:41]: what do you think about the second test?

**Review by botany-demo — APPROVED** [review]
looks good to me

**Review by botany-demo — APPROVED** [review]
ok now it is 💯

**Comment by botany-lead** [comment]
thanks for the help @botany-demo 🥇

**Comment by botany-demo** [comment]
no problem @botany-lead

**Pull request merged**